corehq/apps/export/static/export/js/models.js

```diff
@@ -486,6 +486,7 @@ hqDefine('export/js/models.js', function () {
         });
 
         self.showAdvanced = ko.observable(false);
+        self.showDeleted = ko.observable(false);
```

Comment:
yeah, this model is used very infrequently. it's used when we enabled the feature flag to allow users to define their own columns/tables. When you create your own table, this is the model that gets created